anatomy/tracks/config-json.md

```diff
@@ -175,6 +169,46 @@ Tags are specified in the top-level `tags` field which is defined as an array of
 
 - `typing/static`: the language uses static typing
 - `typing/dynamic`: the language uses dynamic typing
+- `typing/strong`: the language uses strong typing
+- `typing/weak`: the language uses weak typing
+
+## Execution mode
+
+- `execution_mode/compiled`: code is compiled first before being executed
+- `execution_mode/interpreted`: code is interpreted directly
+
+## Platform
+
+- `platform/windows`: runs on Windows
+- `platform/mac`: runs on Mac
+- `platform/linux`: runs on Linux
+- `platform/ios`: runs on iOS
+- `platform/android`: runs on Android
+- `platform/web`: runs on Web
+
+## Runtime
+
+- `runtime/standalone_executable`: runs as standalone executable
+- `runtime/language_specific`: runs on language-specific runtime
+- `runtime/clr`: runs on Common Language Runtime (.NET)
+- `runtime/jvm`: runs on JVM (Java)
+- `runtime/beam`: runs on BEAM (Erlang)
+
+## Used for
```

Comment:
Should these be things that _most_ people use it for or that it _can_ be used for?

Reply:
That's a bit of a judgment call, but I would lean more towards the former than the latter. Virtually all programming languages can be used for scientific computations, but few are really built for that purpose.

Reply:
Yeah think of it as "I'd like to do machine learning, what language should i pick".